setup.py

```diff
@@ -14,7 +14,7 @@
     packages=find_packages(),
     include_package_data=True,
     url="https://github.com/NeurodataWithoutBorders/nwbinspector",
-    install_requires=["pynwb", "natsort", "click", "PyYAML", "jsonschema", "dandi>=0.39.2", "tqdm"],
+    install_requires=["pynwb", "natsort", "click", "PyYAML", "jsonschema", "tqdm"],
```

Comment:
minor: I would recommend to formalize as `extra_requires={'dandi': ["dandi>=0.39.2"]}` (or whatever format it is) and then use `pip install -e .[dandi]` in your workflow.

Reply:
Not a bad idea, we use this on the CT for including non-minimal reqs